ModelBundle/Document/Node.php

```diff
@@ -161,9 +161,17 @@ class Node implements NodeInterface
     protected $role;
 
     /**
+     * @var AreaInterface
+     *
+     * @ODM\EmbedOne(targetDocument="OpenOrchestra\ModelInterface\Model\AreaInterface")
+     */
+    protected $rootArea;
```

Comment:
This property makes a simple container area in a container node. Maybe could you remove this level?